### PR TITLE
fixed issue#11381

### LINKED ISSUE #11381: [UI] Provider-UI: React Hook useEffect warning during build
### Current Behavior
While building the ui-provider, the following warning message is displayed.

### Desired Behavior
No warning message.

### Screenshots/Logs
```
./components/Provider.js
80:6  Warning: React Hook useEffect has a missing dependency: 'loadProvidersFromServer'. Either include it or remove the dependency array.  react-h
```

### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Venki1402**
**Notes for Reviewers**

This PR fixes #11381

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: provider-ui/components/Provider.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export default function Provider() {
 
   useEffect(() => {
     loadProvidersFromServer();
-  }, []);
+  }, [loadProvidersFromServer]);
```
Comment (sudhanshutech) [inline provider-ui/components/Provider.js:80]: @Venki1402 have you ensured that there is no re rendering happening after including this dependency?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Thanks for jumping in, @Venki1402!

File: provider-ui/components/Provider.js (hunk 1 above)
Comment (leecalcote) [reply provider-ui/components/Provider.js:80]: @Venki1402, does this make sense?

**Comment by Ashparshp** [comment]
@Venki1402, let's discuss this PR during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by Ashparshp** [comment]
@Venki1402 Any luck? or
let's discuss this PR during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by sudhanshutech** [comment]
@Venki1402 checking in again after a month...

**Comment by Ashparshp** [comment]
@Venki1402, as we haven't heard back from you, I've unassigned you. Thank you!

**Pull request closed without merging**